src/js/components/DataTable/Body.js

```diff
@@ -216,6 +251,10 @@ const Body = forwardRef(
               return (
                 <Row
                   key={index}
+                  tableRef={ref}
```

Comment:
Is this really the "table" ref? It looks more like the "tbody" ref.

Reply:
Renamed this to `tableBodyRef` to be more accurate in the newly-separated PR for DataTable in https://github.com/grommet/grommet/pull/5397